tools/export/uvision/__init__.py

```diff
@@ -171,7 +171,7 @@ def format_flags(self):
             '--cpreproc --cpreproc_opts=-D__ASSERT_MSG,' +
             ",".join("-D{}".format(s) for s in
                      self.toolchain.get_symbols(for_asm=True)))
-        flags['asm_flags'] = asm_flag_string
+        flags['asm_flags'] = "{} {}".format(" ".join(self.toolchain.flags['asm']), asm_flag_string)
```

Comment:
remove this patch from PR => followed by #10424

Reply:
Done